src/inputs/date.html

```diff
@@ -1,21 +1,41 @@
 <script type="text/ng-template" id="wgn-config-input-date">
-<!-- UI Bootstrap Datepicker Popup directive https://angular-ui.github.io/bootstrap/ -->
+<!--
+	Zengine znDatetimepickerWrapper directive:
+	https://zenginehq.github.io/developers/plugins/api/directives/#zndatetimepickerwrapper
+
+	UI Bootstrap Datepicker Popup directive:
+	http://angular-ui.github.io/bootstrap/versioned-docs/0.12.1/#/datepicker
+-->
 
 	<div ng-controller="wgnDateInputCtrl">
 
-		<input
-			class="form-control"
-			datepicker-options="{{picker.settings}}"
-			datepicker-popup="{{picker.format}}"
-			is-open="picker.opened"
-			maxlength="{{picker.format.length}}"
-			minlength="{{picker.format.length}}"
-			ng-change="setModel($event)"
-			ng-focus="picker.open($event)"
-			ng-model="picker.date"
-			type="text"
-		/>
+		<div zn-datetimepicker-wrapper ng-if="!field.options" ng-model="editing.config[field.id]">
+			<input
+				class="form-control"
+				datepicker-options="dateOptions"
```

Comment:
does it not work to use picker.settings here?

Reply:
I found that I could use conditionals for all of the configuration options. The problem was that because the model for the config (editing.config[field.id]) is modified by the znDatetimepicker directive and the model on the input (date) are not available to the external controller, I can't format the value before it gets to firebase. So, I can set 'yyyy' and view year mode, but the date still get's saved as 'yyyy-MM-dd'. I tried to use a condition for the model on the input, but that seems to be a limitation for the framework.